Sources/MapboxMaps/Foundation/Tracing.swift

```diff
@@ -0,0 +1,98 @@
+import Foundation
+import os.signpost
+import MapboxCommon
+
+private let subsystem = "com.mapbox.maps"
+
+internal extension OSLog {
+    @available(iOS 12, *)
+    private static let _poi = OSLog(subsystem: subsystem, category: .pointsOfInterest)
+    private static let _platform = OSLog(subsystem: subsystem, category: "platform")
+    private static let tracingEnabled: Bool = SettingsServiceFactory.getInstance(storageType: .nonPersistent).isTracingEnabled
+
+    static let platform: OSLog = tracingEnabled ? _platform : .disabled
```

Comment:
Regarding the naming I personally would prefer to see something more project name related like `.map` or `.mmap`